src/client/service/Analyser.js

```diff
@@ -81,8 +81,8 @@ Analyser.prototype.getTitle = function(controller, params)
  */
 Analyser.prototype.sendPageView = function(page, title)
 {
-    ga('send', 'pageview', {
-        page: page,
-        title: title
-    });
+    // ga('send', 'pageview', {
+    //     page: page,
+    //     title: title
+    // });
```

Comment:
@Tom32i i don't kown the best way to desactivate, just remove service declaration in app.js ?